assets/js/base/components/select/style.scss

```
@@ -59,11 +59,19 @@
 			text-transform: none;
 			white-space: nowrap;
 			width: 100%;
+			.has-dark-styles & {
+				background-color: rgba(#000, 0.1);
+				border: 1px solid $input-border-dark;
```

If we are only changing the border color (while keeping border width and style), could we just do:
```suggestion
				border-color: $input-border-dark;
```
This way, if we ever need to change the border width, it will only need to be changed in a single place.